vendors/mediatek/boards/mt7697hx-dev-kit/aws_demos/config_files/aws_defender_config.h

```diff
@@ -0,0 +1,126 @@
+/*
```

Comment:
I believe for consistency and symmetry with the new shadow demo, we should call this file "defender_demo_config.h"

Reply:
addressed